compiler/optimizer/parallelize.go

```diff
@@ -203,16 +203,12 @@ func insertMerge(seq *dag.Sequential, layout order.Layout) error {
 	if layout.IsNil() {
 		return nil
 	}
-	var reverse bool
-	if layout.Order == order.Desc {
-		reverse = true
-	}
 	// XXX Fix this to handle multi-key merge; change dag.Merge.Reverse.
 	// See Issue #2657.
```

Comment:
```suggestion
```

Reply:
Multi-key merge is still not possible with this pr so the comment should remain.